cpp/modmesh/buffer/pymod/TypeBroadcast.hpp

```diff
@@ -40,11 +40,9 @@ namespace python
 template <typename T /* original type */, typename D /* for destination type */>
 struct TypeBroadcastImpl
 {
-    using slice_type = small_vector<int>;
```

Comment:
It's OK to have type aliases in the class.  Why removing them?

Reply:
I misunderstood. I thought you were asking to remove all aliases.

Reply:
reverted.

Reply:
No problem